NitroxClient/Communication/Packets/Processors/StorageSlotAddItemProcessor.cs

```diff
@@ -0,0 +1,36 @@
+﻿using NitroxClient.Communication.Abstract;
+using NitroxClient.Communication.Packets.Processors.Abstract;
+using NitroxClient.GameLogic;
+using NitroxClient.GameLogic.Helper;
+using NitroxClient.GameLogic.Spawning;
+using NitroxModel.DataStructures.GameLogic;
+using NitroxModel.Logger;
+using NitroxModel.Packets;
+using UnityEngine;
+
+namespace NitroxClient.Communication.Packets.Processors
+{
+    class StorageSlotAddItemProcessor : ClientPacketProcessor<StorageSlotItemAdd>
+    {
+        private readonly IPacketSender packetSender;
+        private readonly StorageSlots storageSlots;
+
+        public StorageSlotAddItemProcessor(IPacketSender packetSender, StorageSlots storageSlots)
+        {
+            this.packetSender = packetSender;
+            this.storageSlots = storageSlots;
+        }
+
+        public override void Process(StorageSlotItemAdd packet)
+        {            
+            ItemData itemData = packet.ItemData;
+            GameObject item = SerializationHelper.GetGameObject(itemData.SerializedData);
+
+            // Mark this entity as spawned by the server
+            item.AddComponent<NitroxEntity>();
```

Comment:
Will this add multiple of the same component if a player stores it, picks it up, then stores it again?

Reply:
I "stole" that piece of code from ItemContainerAddProcessor. But we can test that. If it does, then we can change it on both parts

Reply:
I testet it and it seems the NitroxEntity component will not be serialized with the item.
Nontheless i build a statement around it so it will never add a second NitroxEntity